src/styles/index.scss

```diff
@@ -1,3 +1,125 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap");
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+}
+
+
 body {
   margin: 0;
+  font-family: Roboto, sans-serif;
+  background-color: #fff;
+}
+
+
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+
+  &__title {
+    font-size: 12px;
+    font-weight: 500;
+    margin-bottom: 4px;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &__code {
+    font-weight: 400;
+    font-size: 10px;
+    color: #616070;
```

Comment:
Create variable for repeated color and use

Reply:
Not fixed